setup.py

```diff
@@ -384,8 +384,8 @@ def get_sphinx_theme_version() -> str:
     'pdpyras>=4.1.2,<5',
 ]
 papermill = [
-    'nteract-scrapbook[all]>=0.3.1',
     'papermill[all]>=1.2.1',
+    'scrapbook[all]>=0.3.1',
```

Comment:
Why do we need to pin min version?
`scrapbook` starts from version `0.5.0`

https://pypi.org/project/scrapbook/#history
https://pypi.org/project/nteract-scrapbook/#history

Reply:
Good point!